### ISSUE TITLE
Highlight changes instead of underlining them

### ISSUE THREAD

**Issue by arbitrary-dev** [ymattw/ydiff#99]
Is there a way to highlight changes instead of underlining them?

Because underlining is hard to spot, it'd be nice to have an option to highlight them as is done in case of removals/deletions.

**Comment by ElectricRCAircraftGuy** [comment]
What is the meaning of underlined changes vs highlighted changes anyway?

**Comment by arbitrary-dev** [comment]
Looks better to me:

![areashot-2021-03-20-163909](https://user-images.githubusercontent.com/6984946/111873873-30682580-899b-11eb-8398-40d5a56ae7b6.png)

**Comment by ymattw** [comment]
Only after word-level diffs are provided, switching to reversed color is more human readable than underline for changed text. This is now addressed in commit 5126c52. Check out the latest revision!
